### DISCUSSION TITLE
Dịch purity, impurity

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#39] [Terminology] [answered]
Hai từ này xuất hiện trong chương Decision Tree. Mình có vài option:
* `purity`: `độ tinh khiết`
* `impurity`: `độ lẫn lộn`, `độ pha tạp`

Mọi người thấy sao nhỉ @mlbvn/handson-ml2 ?

**Comment by lkhphuc** [comment]
Hai từ này bên https://github.com/mlbvn/handson-ml2-book/pull/148/files#diff-754c9fa6e12e7320d5e4cd442492c8c4124bf24007e167b818a2b344f8d83547R200 mình cũng đang dịch là `nguyên chất`, `pha tạp`.
Mình thấy đây là concept chứ cũng ko phải từ khoá gì dịch từ nào thuận ngữ cảnh là được.

**Reply by minhduc0711** [reply]
nếu dùng trong `gini impurity` thì cũng thành từ khóa rồi, với cả mình thích dịch nhất quán hơn (trong context của decision tree)

**Reply by lkhphuc** [reply]
chỉ có `gini` là function coefficient thôi mà nhỉ.

**Reply by ngcthuong** [reply]
Gini impunity nó khác với Gini coeficient mà nhỉ? 
Mà hình như Gini là tên nhà toán học, cần viết hoa. ,

**Comment by minhduc0711** [comment, marked as answer]
tạm chốt nhé mọi người
* `purity`: `độ tinh khiết`
* `impurity`: `độ pha tạp`
